engines/wintermute/base/sound/base_sound_buffer.cpp

```diff
@@ -107,7 +107,7 @@ bool BaseSoundBuffer::loadFromFile(const Common::String &filename, bool forceRel
 	#ifdef USE_VORBIS
 		if (strFilename.hasSuffix(".ogg")) {
 			_stream = Audio::makeVorbisStream(_file, DisposeAfterUse::YES);
-		}
+		} else
```

Comment:
Actually, two things are off here: 

* The indentation shouldn't have been increased in PR808
* This leads to silently skipping .ogg-files. Not sure I like that, at the very least the macro-check should be within the if, but IMHO we should at the very least fire off a warning() in this case, if not an error().

Reply:
Ad 1) Totally correct.
Ad 2) In practise it won't because of the change introduced [here](https://github.com/scummvm/scummvm/blob/d022572f1624fd2d1ccc0dbb53ab5bda17b50e85/engines/wintermute/wintermute.cpp#L150), games which require Ogg Vorbis won't load if it's not compiled in, no?

Reply:
Right, in that case I'd prefer an error(), as the case where you'd enter here would be a game that _has_ the lowres flag set, while still having a .ogg resource. In which case the somewhat confusing "Unknown filetype"-error message" would be more confusing than helpfull.

Reply:
Indeed. Fixed in commit 98067b0fe6cdccd6a1cbc6abda8fa36a39bb7fd7, if you'd prefer a more verbose error message feel free to change it yourself.